apps/retrospective/services/sprint_task.go

```diff
@@ -53,6 +54,11 @@ func (service SprintTaskService) List(
 		if task.IsTrackerTask {
 			task.URL = connection.GetTaskUrl(task.Key)
 		}
+		var participants_slice = strings.Split(task.TaskParticipants, ", ")
+
+		if len(participants_slice) > 1 {
```

Comment:
We should handle empty slices in the utility itself.

Reply:
It's already handled in the function itself. Better to remove this from here.